client/src/components/Playlist.tsx

```diff
@@ -194,12 +189,13 @@ function Playlist({
     // update spotify playlist with current slots
     // convert each slot to a spotify uri of a track
     const uris = await Promise.all(slots.map(async (slot) => {
-      const { type, name, pool_id, pool_spotify_id } = slot;
+      const { id: slotId, type, name, pool_id, pool_spotify_id } = slot;
```

Comment:
Accidentally left these edits in -- I was originally working on the ability to dynamically update the playlist when I got sidetracked by the auth flow.